source/reference/mongosync/mongosync-behavior.txt

```diff
@@ -113,6 +113,15 @@ destination cluster.
 Chunk Distribution
 ''''''''''''''''''
 
+.. important::
+   
+   Even if the source cluster is balanced, ``mongosync`` does not
+   ensure balance of the destination cluster. Because ``mongosync``
+   does not support the execution of sharding operations during
+   migration, you must wait until it is safe to accept writes 
```

Comment:
can we add the link to [Limitations](https://www.mongodb.com/docs/cluster-to-cluster-sync/current/reference/limitations/) from this [comment](https://github.com/mongodb/docs-cluster-to-cluster-sync/pull/678#discussion_r2012304938)

once that change is added I am good to approve, thank you!